python/pyspark/sql/tests/test_udf_profiler.py

```diff
@@ -51,6 +51,7 @@ def add2(x):
     action(df)
 
 
+@unittest.skipIf(not have_pyarrow, pyarrow_requirement_message)
```

Comment:
cc @xinrong-meng Mind double checking? It seems weird that it needs Arrow here.

Reply:
Thanks Hyukjin.
@panbingkun Would you please skip the test "UDFProfilerTests.test_unsupported" if no Arrow installed, instead of skipping the whole class?

Reply:
Okay